Lesson4.ipynb

```diff
@@ -29,6 +33,1310 @@
     "\n",
     "Select **Lesson4.ipynb**"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Read csv file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What is csv file?\n",
+    "\n",
+    "A CSV is a comma separated values file which allows data to be saved in a table structured format. CSVs are a kind of spreadsheet same with Excel but with a .csv extension. Traditionally they take the form of a text file containing information separated by commas, hence the name.\n",
+    "\n",
+    "** Content in CSV Sample **\n",
+    "\n",
+    "The followings are the part of data from Biggest Loser:\n",
```

Comment:
Replace "The followings are the part of data from Biggest Loser:"
with 
"The following sample data is from the Biggest Loser step challenge. For those unfamiliar with the Biggest Loser step challenge, this was a 4-week long challenge where teams (each consisting 5 members) combined their daily step count in order to reach these goals: 
1) to be the first team to reach 1 million steps
2) to have the highest overall step count amongst all teams
3) to show the most improved team step count (from weeks 1+2 to weeks 3+4)